railties/test/generators_test.rb

```diff
@@ -34,6 +34,18 @@ def test_generator_suggestions
     assert_match "Maybe you meant 'migration'", output
   end
 
+  def test_generator_suggestions_except_en_locale
+    orig_available_locales = I18n.available_locales
+    orig_default_locale = I18n.default_locale
+    I18n.available_locales = :ja
+    I18n.default_locale = :ja
+    name = :migrationz
+    assert_nothing_raised { quietly { Rails::Generators.invoke name } }
```

Comment:
Thanks for the PR!

Is there something else that can be tested other than that nothing was raised? Should this do what the above test is doing? `assert_match "Maybe you meant 'migration'", output` ?

In general it's better to test that the behavior is correct rather than that it doesn't raise.